src/main/app/src/pages/BrowsePage.js

```diff
@@ -0,0 +1,100 @@
+import React, { useState, useEffect } from 'react';
+import 'bulma/css/bulma.css';
+import '../css/home.css';
+
+// This is a stateless, functional React component used to render each resource in a card format. 
+const BrowseCard = ({ title, url }) => {
+  if (!title) return <div />
+  return (
+    <div className = "p-3">
+      <div className="card">
+        <div className="card-content">
+          <h1 className="title">{title}</h1>
+          <button className="button is-small is-rounded" href={url}>View more</button>
+          <button className="button is-small is-rounded">schedule event</button>
+        </div>
+      </div>
+    </div>
+  );
+};
+
+// Web Servlet links.
+const articleData = './articleData';
+const videoData = './videoData';
+const gameData = './gameData';
+
+/* Component for browse page */
+const BrowsePage = () => {
+
+  // Create initial state for activity selection with Games as default.
+  const [activity, updateActivity] = React.useState("games");
+
+  // Create initial state for web servlet links with gameData servlet as default.
+  const [currentData, updateDataLink] = React.useState(gameData);
+
+  // Update activty selection and web servlet state based on dropdown.
+  const handleActivitySelection = evt => {
+    updateActivity(evt.target.value);
+    if (evt.target.value == "Reading") {
+      updateDataLink(articleData);
+    } else if (evt.target.value == "Active") {
+      updateDataLink(videoData);
+    } else if (evt.target.value == "Games") {
+      updateDataLink(gameData);
+    }
+  }
+
+  const [links, updateLinks] = React.useState([]);
+  useEffect(() => {
+    console.log(activity);
```

Comment:
Remove debug code.

Reply:
I kept this so that we can see that the web servlet being fetched matched the activity cause there's still more code that needs to be added to optimize this function.

Reply:
We will work on this together later. I understand why you kept it for now.

Reply:
Add TODO comments in cases like this.